iframe-embed.html

```diff
@@ -0,0 +1,16 @@
+<iframe title="{ein Bildtitel o.ä.}" loading="lazy" width="100%" height="600" id="rbb-data--{project-name}" src="https://www.rbb-online.de/static/rbb/rbb-data/{project-name}/"></iframe>
+<style>
+  #rbb-data--{project-name} {
+    width: 1px;
+    min-width: 100%;
```

Comment:
i know that this tripped me up in the past because i  had no idea where the values came from. Maybe it makes sense to state that they're from the resizer's repo, what do you think @sophiamersmann ?

Reply:
Agreed ✌🏻